SQL/0000-00-00-schema.sql

```diff
@@ -72,6 +72,15 @@ CREATE TABLE `language` (
 INSERT INTO language (language_code, language_label) VALUES
     ('en-CA', 'English');
 
+CREATE TABLE `sex` (
+  `SexID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `SexValue` varchar(255) NOT NULL,
```

Comment:
Nitpicky but as I said during the meeting, I find "SexValue" to be a little generic, and think something like "SexName" would be more descriptive (this also applies to the rest of this PR obviously).

Reply:
Updated "SexValue" to "Sex", and also modified charset to utf8mb4